src/Platform/Microsoft.Testing.Platform/CommandLine/PlatformCommandLineProvider.cs

```diff
@@ -111,7 +111,7 @@ public Task<ValidationResult> ValidateOptionArgumentsAsync(CommandLineOption com
         {
             string arg = arguments[0];
             int size = arg.Length;
-            if ((char.ToLowerInvariant(arg[size - 1]) != 'h' && char.ToLowerInvariant(arg[size - 1]) != 'm' && char.ToLowerInvariant(arg[size - 1]) != 's') || !float.TryParse(arg[..(size - 1)], out float _))
+            if ((char.ToLowerInvariant(arg[size - 1]) != 'h' && char.ToLowerInvariant(arg[size - 1]) != 'm' && char.ToLowerInvariant(arg[size - 1]) != 's') || !float.TryParse(arg[..(size - 1)], CultureInfo.InvariantCulture, out float _))
```

Comment:
src/Platform/Microsoft.Testing.Platform/CommandLine/PlatformCommandLineProvider.cs(114,167): error CS7036: (NETCORE_ENGINEERING_TELEMETRY=Build) There is no argument given that corresponds to the required parameter 'result' of 'float.TryParse(string, NumberStyles, IFormatProvider, out float)'